libraries/stm32f0xx/src/retarget.c

```diff
@@ -0,0 +1,34 @@
+#include "retarget.h"
```

Comment:
Yeah, I was originally going to remove this and replace it with the UART code, but I think I'll just merge this in and replace it with UART later.